.pre-commit-config.yaml

```diff
@@ -46,7 +46,7 @@ repos:
         # mirror does not pull tags for old major versions once a new major
         # version tag is published.
```

Comment:
Can you add some of the details from the PR description to this comment?  For instance, in the PR description you mention "errors" but here you just mention that v4 is "unstable".

Reply:
Nevermind.  My comment is OBE after #186.